diff -Naur old/config.h new/config.h
--- old/config.h	2009-01-23 13:44:44.000000000 +0000
+++ new/config.h	2009-01-23 13:44:59.000000000 +0000
@@ -67,7 +67,7 @@
 /* #undef G_ATOMIC_IA64 */
 
 /* powerpc atomic implementation */
-#define G_ATOMIC_POWERPC 1
+/* #define G_ATOMIC_POWERPC 1 */
 
 /* s390 atomic implementation */
 /* #undef G_ATOMIC_S390 */
diff -Naur old/libtool new/libtool
--- old/libtool	2009-01-23 13:44:51.000000000 +0000
+++ new/libtool	2009-01-23 13:45:02.000000000 +0000
@@ -322,7 +322,7 @@
 link_all_deplibs=yes
 
 # Compile-time system search path for libraries
-sys_lib_search_path_spec="/usr/lib/powerpc-apple-darwin9/4.0.1 /usr/lib /usr/lib/gcc/powerpc-apple-darwin9/4.0.1 /usr/local/lib"
+sys_lib_search_path_spec="/Developer/SDKs/MacOSX10.4u.sdk/usr/lib /usr/local/lib"
 
 # Run-time system search path for libraries
 sys_lib_dlsearch_path_spec="/usr/local/lib /lib /usr/lib"
@@ -7557,7 +7557,7 @@
 link_all_deplibs=yes
 
 # Compile-time system search path for libraries
-sys_lib_search_path_spec="/usr/lib/powerpc-apple-darwin9/4.0.1 /usr/lib /usr/lib/gcc/powerpc-apple-darwin9/4.0.1 /usr/local/lib"
+sys_lib_search_path_spec="/Developer/SDKs/MacOSX10.4u.sdk/usr/lib /usr/local/lib"
 
 # Run-time system search path for libraries
 sys_lib_dlsearch_path_spec="/usr/local/lib /lib /usr/lib"
